.github/workflows/maven.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
```

Comment:
Please rebase this branch with the main branch so this file wouldn't show up on the PR anymore